.eslintrc

```diff
@@ -8,6 +8,10 @@
     "ecmaVersion": 2017,
     "sourceType": "module"
   },
+  "rules": {
+    // Fixes eslint no-invalid-this error for fat arrow class methods
+    "no-invalid-this": 0
```

Comment:
Where was this flagging? Wondering if it's something we should turn off for now but look to fix + turn back on.

Reply:
It was flagging the usage of `this` in the `onChange`, `onClick` and `onKeyUp` in the `EmailForm` component. 
https://github.com/keep-network/website/blob/5afdd75c1a8072fa7615283e84390e1535d654ce/src/components/EmailForm.js#L53, https://github.com/keep-network/website/blob/5afdd75c1a8072fa7615283e84390e1535d654ce/src/components/EmailForm.js#L75
https://github.com/keep-network/website/blob/5afdd75c1a8072fa7615283e84390e1535d654ce/src/components/EmailForm.js#L80

I think it's nice to leave on so we can use arrow functions for instance methods on a class, but maybe there's a better way to make this pass the linter. However, I'd like to refactor this component to use react hooks instead anyhow.

Reply:
That refactoring is probably the Ideal™ fix. Cool cool!